<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FitsImporter</class>
 <widget class="QWizard" name="FitsImporter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import FITS files</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/images/applications-astronomy.png</normaloff>:/icons/images/applications-astronomy.png</iconset>
  </property>
  <widget class="ExtWizardPage" name="wizardPageLoadFiles">
   <property name="title">
    <string>Setup files</string>
   </property>
   <property name="subTitle">
    <string>Select the different files for the channels. Optionally, they can be aligned.</string>
   </property>
   <attribute name="pageId">
    <string notr="true">21</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_4" columnstretch="0,1,0">
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditRed">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>H-alpha (optional)</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEditH">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButtonGreen">
      <property name="text">
       <string>Select...</string>
      </property>
      <property name="icon">
       <iconset theme="document-open">
        <normaloff>.</normaloff>.</iconset>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEditBlue">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Channel 1 (Red)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButtonBlue">
      <property name="text">
       <string>Select...</string>
      </property>
      <property name="icon">
       <iconset theme="document-open">
        <normaloff>.</normaloff>.</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButtonRed">
      <property name="text">
       <string>Select...</string>
      </property>
      <property name="icon">
       <iconset theme="document-open">
        <normaloff>.</normaloff>.</iconset>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Luminosity (optional)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Channel 2 (Green)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="pushButtonLuminosity">
      <property name="text">
       <string>Select...</string>
      </property>
      <property name="icon">
       <iconset theme="document-open">
        <normaloff>.</normaloff>.</iconset>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditGreen">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Channel 3 (Blue)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEditLuminosity">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="3">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="5">
        <widget class="QPushButton" name="pushButtonClockwise">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Rotate selected image 180Â°&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Rotate</string>
         </property>
         <property name="icon">
          <iconset theme="object-rotate-right">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButtonReset">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Reset</string>
         </property>
         <property name="icon">
          <iconset theme="chronometer-reset">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButtonLoad">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Load Previews</string>
         </property>
         <property name="icon">
          <iconset theme="upload-media">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="shortcut">
          <string>Ctrl+R</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="pushButtonPreview">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Shows a larger preview of selected image</string>
         </property>
         <property name="text">
          <string>Preview</string>
         </property>
         <property name="icon">
          <iconset theme="document-preview">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="6" column="0" rowspan="3" colspan="3">
     <widget class="QGroupBox" name="grpPreviews">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>700</width>
        <height>130</height>
       </size>
      </property>
      <property name="title">
       <string>Previews</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="verticalLayoutPreviews"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="pushButtonH">
      <property name="text">
       <string>Select...</string>
      </property>
      <property name="icon">
       <iconset theme="document-open">
        <normaloff>.</normaloff>.</iconset>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="3">
     <widget class="QGroupBox" name="alignGroupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>Alignment</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout1">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="5" column="1">
          <widget class="QCheckBox" name="autoCropCheckBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Auto-crop</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QRadioButton" name="mtb_radioButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>&amp;MTB</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QRadioButton" name="ais_radioButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Hugin's align_&amp;image_stack</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>1</width>
             <height>1</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QCheckBox" name="alignCheckBox">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>&amp;Autoalign images</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="ExtWizardPage" name="wizardPageAlignment">
   <property name="title">
    <string>Alignment</string>
   </property>
   <property name="subTitle">
    <string>The alignment of the input files is in progress. Please be patient!</string>
   </property>
   <attribute name="pageId">
    <string notr="true">22</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Progress:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBoxMessages_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Messages</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QTextEdit" name="textEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPagePreview">
   <property name="title">
    <string>Preview</string>
   </property>
   <property name="subTitle">
    <string>Please fine-tune the color part of the channels.</string>
   </property>
   <attribute name="pageId">
    <string notr="true">99</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_5" rowstretch="0,0,0,0">
    <item row="2" column="0">
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>Blue</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QDoubleSpinBox" name="dsbRedRed">
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSlider" name="hsRedRed">
      <property name="maximum">
       <number>10000</number>
      </property>
      <property name="value">
       <number>10000</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>Green</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QDoubleSpinBox" name="dsbBlueBlue">
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="4">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Preview</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="0" column="0">
        <widget class="QLabel" name="previewLabel">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>200</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="lineWidth">
          <number>3</number>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSlider" name="vsGamma">
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="singleStep">
          <number>100</number>
         </property>
         <property name="pageStep">
          <number>1000</number>
         </property>
         <property name="value">
          <number>2200</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::NoTicks</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_17">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Red</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QDoubleSpinBox" name="dsbGreenGreen">
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSlider" name="hsBlueBlue">
      <property name="maximum">
       <number>10000</number>
      </property>
      <property name="value">
       <number>10000</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSlider" name="hsGreenGreen">
      <property name="maximum">
       <number>10000</number>
      </property>
      <property name="value">
       <number>10000</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ExtWizardPage</class>
   <extends>QWizardPage</extends>
   <header>UI/ExtWizardPage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditRed</tabstop>
  <tabstop>pushButtonRed</tabstop>
  <tabstop>lineEditGreen</tabstop>
  <tabstop>pushButtonGreen</tabstop>
  <tabstop>lineEditBlue</tabstop>
  <tabstop>pushButtonBlue</tabstop>
  <tabstop>lineEditLuminosity</tabstop>
  <tabstop>pushButtonLuminosity</tabstop>
  <tabstop>lineEditH</tabstop>
  <tabstop>pushButtonH</tabstop>
  <tabstop>pushButtonLoad</tabstop>
  <tabstop>pushButtonReset</tabstop>
  <tabstop>pushButtonPreview</tabstop>
  <tabstop>pushButtonClockwise</tabstop>
  <tabstop>alignCheckBox</tabstop>
  <tabstop>ais_radioButton</tabstop>
  <tabstop>mtb_radioButton</tabstop>
  <tabstop>autoCropCheckBox</tabstop>
  <tabstop>textEdit</tabstop>
  <tabstop>hsRedRed</tabstop>
  <tabstop>dsbRedRed</tabstop>
  <tabstop>hsGreenGreen</tabstop>
  <tabstop>dsbGreenGreen</tabstop>
  <tabstop>hsBlueBlue</tabstop>
  <tabstop>dsbBlueBlue</tabstop>
  <tabstop>vsGamma</tabstop>
 </tabstops>
 <resources>
  <include location="../../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>alignCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>ais_radioButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>501</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>501</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>alignCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>mtb_radioButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>501</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>501</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>alignCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>autoCropCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>501</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>526</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
